### PR TITLE
Added permissions requirements to file uploads

### LINKED ISSUE #79: Add permissions requirements for sending SDLArtworks
Add note about the `hmiLevel` needed to send `SDLArtwork` RPCs to the *Uploading Files and Graphics* section.

### PR DESCRIPTION
**Pull request by NicoleYarroch**
Fixes #79 

This pull request **adds new content**.

## Summary of Changes
* Added information that images can not be sent when the `hmiLevel` is `NONE`.
* Relocated *Uploading Files and Graphics* section to the *Displaying Information* section.
* Fixed broken links.
* Updated a Swift `SDLArtwork` code example.

### CODE REVIEW

File: docs/Displaying Information/Uploading Files and Graphics/index.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ## Uploading Files and Graphics
-You should be aware of these three things when using images:
+You should be aware of these four things when using images in your SDL app:
 
 1. You may be connected to a head unit that does not have the ability to display images.
 1. You must upload images from your mobile device to the head unit before using them in a template.
 1. Persistant images are stored on a head unit between sessions. Ephemeral images are destroyed when a sessions ends.
```
Comment (joeljfischer) [inline docs/Displaying Information/Uploading Files and Graphics/index.md]: "sessions ends" -> "session ends (when the user turns off their vehicle)"

File: docs/Displaying Information/Uploading Files and Graphics/index.md [hunk 2]
```diff
@@ -161,7 +162,7 @@ NSUInteger bytesAvailable = self.sdlManager.fileManager.bytesAvailable;
 let bytesAvailable = sdlManager.fileManager.bytesAvailable
 ```
 
-### Check if a File Has Already Been Uploaded
+### Check if a File has Already Been Uploaded
```
Comment (joeljfischer) [inline docs/Displaying Information/Uploading Files and Graphics/index.md]: "Has" should be capitalized here.

Comment (NicoleYarroch) [reply docs/Displaying Information/Uploading Files and Graphics/index.md]: Fixed.

File: docs/Displaying Information/Uploading Files and Graphics/index.md (hunk 1 above)
Comment (NicoleYarroch) [reply docs/Displaying Information/Uploading Files and Graphics/index.md]: Fixed

**Pull request merged**